Sources/MeiliSearch/Request.swift

```diff
@@ -28,6 +36,30 @@ public final class Request {
     self.session = config.session ?? URLSession.shared
   }
 
+  private func request(
+    _ httpMethod: HTTPMethod,
+    _ url: URL,
+    _ data: Data?,
+    _ headers: [String: String] = [:]
```

Comment:
if we use named variables we can avoid adding `null` as a parameter in our other function calls. I think also that optional fields should always be the last parameters. 
```suggestion
    _ headers: [String: String] = [:]
    _ data: Data?,
```